frontend/src/redux/headlampEventSlice.ts

```diff
@@ -0,0 +1,452 @@
+import {
```

Comment:
Can you please add a headlampEventSlice.test.ts with some minimal tests to cover adding a callback and dispatching an event?

Reply:
Yup. Sorry I missed this.